docs/docs/3-features-modules/9-brevo-module/index.md

```diff
@@ -1,3 +1,5 @@
 ---
 title: Brevo Module
 ---
+
+The Comet Brevo Module integrates Brevo (formerly Sendinblue) with your Comet project and allows you to create and send newsletters and transactional emails. It also enables you to add and manage contacts, as well as custom target groups and recipient lists, directly in the Comet admin interface.
```

Comment:
IMO we could drop the second sentence since that's what I'd expect from a newsletter tool.

```suggestion
The Comet Brevo Module integrates Brevo (formerly Sendinblue) with your Comet project and allows you to create and send newsletters and transactional emails.
```